nima_core/storage/process_guard.py

```
@@ -0,0 +1,220 @@
+    
+    try:
+        pid = int(PID_FILE.read_text().strip())
+        # Check if process is actually running
+        try:
+            os.kill(pid, 0)
+            # Process is running - check if it's actually a gateway
+            try:
+                cmdline = Path(f"/proc/{pid}/cmdline").read_text().replace('\0', ' ')
+                if 'openclaw' in cmdline or 'gateway' in cmdline:
+                    return []  # Not a zombie - active gateway
+            except:
```

Ruff (0.15.5)

[error] Do not use bare `except`

(E722)